packages/amplify-e2e-tests/schemas/transformer_migration/auth-model-v1.graphql

```diff
@@ -0,0 +1,28 @@
+type Post @model @auth(rules: [{ allow: owner }]) {
+  id: ID!
+  title: String!
+  createdAt: AWSDateTime
+  updatedAt: AWSDateTime
+  owner: String
+}
+
+type PostPublic @model @auth(rules: [{ allow: public }]) {
+  id: ID!
+  title: String
+}
+
+type PostPublicIAM @model @auth(rules: [{ allow: public, provider: iam }, { allow: private, provider: iam }]) {
```

Comment:
nit: can we change the name to show its using IAM?
```suggestion
type PostIAM @model @auth(rules: [{ allow: public, provider: iam }, { allow: private, provider: iam }]) {
```

Reply:
This model is used to valid public iam rule only. So I've removed the private rule.